CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `internal_pointer_mut()` function on `QModelIndex`
 - `c_void` in CXX-Qt-lib for easy access to `void *`
 - `CxxQtThread` is now marked as `Sync` so that it can be used by reference
+- Add extra lib which contains: `QCommandLineOption`, `QCommandLineParser`, `QElapsedTimer`
```

Comment:
maybe mention the crate name 

Add `cxx-qt-lib-extras` crate which contains: `QCommandLineOption`, `QCommandLineParser`, `QElapsedTimer`